cmp/grid/GridModel.js

```diff
@@ -191,7 +198,10 @@ export class GridModel {
         this.emptyText = emptyText;
         this.rowClassFn = rowClassFn;
         this.groupSortFn = withDefault(groupSortFn, this.defaultGroupSortFn);
-        this.contextMenuFn = withDefault(contextMenuFn, this.defaultContextMenuFn);
+        this.contextMenu = withDefault(this.createContextMenuFn(contextMenu), this.defaultContextMenuFn());
+
+        // Deprecation warning added as of 28.2 - remove in future major version.
```

Comment:
Doesn't look like we are *actually* respecting/using contextMenuFn here.  Probably better to just cleanly consider this a breaking change in how we describe it, rather than a deprecation.  But the warning is good.

At this point, I think we are probably looking at a v29 release for the main develop branch in any case.